changelogs/unreleased/gh-9924-luajit-fixes.md

```diff
@@ -0,0 +1,17 @@
+## bugfix/luajit
+
+Backported patches from the vanilla LuaJIT trunk (gh-9924). The following issues
+were fixed as part of this activity:
```

Comment:
I would also mention added UBSan support and seems commit "Prevent sanitizer warning in snap_restoredata()." is missing in changelog.

Reply:
In a private conversation decided to not add ubsan entry to changelog.